index.html

```diff
@@ -4,7 +4,71 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link href="style.css" rel="stylesheet" />
     <title>Personal Website</title>
   </head>
-  <body></body>
+
+  <body>
+    <header>
+      <h1>Adriana Morales Tremolada</h1>
+      <h2>Human</h2>
+      <nav>
+        <ul>
+          <li>About me</li>
+          <li>Interests</li>
+          <li>What the future holds</li>
+          <li>Contact</li>
+        </ul>
+      </nav>
```

Comment:
I like that you put the nav links into a list. Very good.

Reply:
🤓